caffe2/perfkernels/embedding_lookup_idx.cc

```diff
@@ -185,10 +185,8 @@ static bool EmbeddingLookupGenericSlowIdx(
       return;                                                                                         \
     }                                                                                                 \
     int64_t current = 0;                                                                              \
-    for (int m = 0; m < output_size; ++m) {                                                           \
-      for (int64_t i = offsets[m];                                                                    \
-           i < (m == output_size - 1 ? index_size : offsets[m + 1]);                                  \
-           ++i) {                                                                                     \
+    for (int m = 0; m < output_size - 1; ++m) {                                                       \
```

Comment:
Why output_size - 1 ? We still want to check OOB indices for m == output_size - 1